nohup: ignoring input
Loading Data:
Traceback (most recent call last):
  File "pandas/_libs/lib.pyx", line 1897, in pandas._libs.lib.maybe_convert_numeric
ValueError: Unable to parse string "open"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "gap_up_stock_finders.py", line 18, in <module>
    stock_price_df['open'] = pandas.to_numeric(stock_price_df['open'])
  File "/home/grads/r/randyardywibowo/.local/lib/python3.6/site-packages/pandas/core/tools/numeric.py", line 151, in to_numeric
    values, set(), coerce_numeric=coerce_numeric
  File "pandas/_libs/lib.pyx", line 1934, in pandas._libs.lib.maybe_convert_numeric
ValueError: Unable to parse string "open" at position 340
